docs/design/2020-02-24-ticdc-mq-protocol-cn.md

```diff
@@ -127,7 +127,44 @@ Value:
 
 ## Kafka 分区策略
 
-partition 数量不限，各个 partition 之间对等。
+* Row Changed Event 根据一定的分区算法分发至不同的 partition
+* Resolved Event 和 DDL Event 广播至所有的 partition
+
+### Row Changed Event 划分 Partition 算法
```

Comment:
What about adding a part to summarize which partition algorithm is used in TiCDC now?